content/en/docs/contributing/release-process.md

```diff
@@ -239,157 +269,154 @@ required only to avoid rate-limits imposed on anonymous API users.
+    2. **(initial alpha, subsequent alpha and beta only)** Tick the box "This is
+       a pre-release".
 
-      {{% pageinfo %}}
-https://github.com/jetstack/cert-manager/releases/tag/v1.0.0 🎉
-      {{% /pageinfo %}}
+    3. Click "Publish" to make the GitHub release live. This will create a Git
+       tag automatically.
 
-### Final Release
+8. Finally, post a Slack message as an answer to the first message. Toggle the
+   check box "Also send to `#cert-manager-dev`" so that the message is well
+   visible. Also cross-post the message on `#cert-manager`.
 
-After releasing one or more alpha and beta releases,
-you will release the final version.
-For the final release, you should follow the process described above with the following changes and additional steps:
+    <div class="pageinfo pageinfo-primary"><p>
+    https://github.com/jetstack/cert-manager/releases/tag/v1.0.0 🎉
+    </p></div>
 
-#### Full Release Notes
+9. Proceed to the post-release steps:
 
-The release notes for the final release should include all changes since the last minor release.
+    1. **(final release only)** Open a PR to
+       [`jetstack/testing`](https://github.com/jetstack/testing) and change Prow's
+       config. To do this, take inspiration from [Maartje's PR
+       example](https://github.com/jetstack/testing/pull/397/files).
 
-#### Rollover Testing Infrastructure
+    2. **(final release only)** Push a new release branch to
+       [`jetstack/cert-manager`](https://github.com/jetstack/cert-manager). If the
+       final release is `v1.0.0`, then push the new branch `release-1.1`:
 
-After releasing the final release you will need to update the testing infrastructure,
-so that it uses the latest release as `release-previous`,
-and you will need to create a new release branch in the cert-manager repository which will be treated as `release-next`,
-and both these branches will be tested periodically.
-
-For example see the PR  [Prepare testing for the cert-manager `v1.0` release](https://github.com/jetstack/testing/pull/397).
-
-#### Rollover Documentation
-
-You will also need to update the versions and branches in the cert-manager website configuration.
+        ```bash
+        # Must be run from the cert-manager repo folder.
+        git checkout -b release-1.1 v1.0.0
+        git push origin release-1.1
+        ```
 
-For example see the PR [Configure website for the `v1.0` release](https://github.com/cert-manager/website/pull/309).
+    3. **(final release only)** Open a PR to
+       [`cert-manager/website`](https://github.com/cert-manager/website) with
+       updates to the website configuration. To do this, take inspiration from
+       [Maartje's PR
+       example](ttps://github.com/cert-manager/website/pull/309/files).
 
 ## Patch Releases
 
-A patch release contains critical bug fixes for the project.  They are managed on
+A patch release contains critical bug fixes for the project. They are managed on
```

Comment:
The link at the bottom - process for releasing a minor version is now broken (after header refactoring?)

`1.` and `2.` are both rendered as `1.` due to markdown parsing

Reply:
Ooops, I need to find a way to automate the check "we shall not break ID anchors".